### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by beth-dixxon**
# Context

Content changes across front-end UI. 
We've added 
- details component, including additional guidance for PP's on the type of ap screen 
- changed content on Type of AP screen radio buttons
- updated guidance information on 'Location factors' screen

No

No

### CODE REVIEW

File: server/views/applications/pages/location-factors/describe-location-factors.njk [hunk 1]
```diff
@@ -13,7 +13,9 @@
           classes: "govuk-label--m"
         },
         hint: {
-          text: "For example SW1."
+          html: '<p>For example SW1.</p>
+          <p>We aim to accommodate someone within 50 miles of the postcode area provided.</p>
+          <p></p>'
```
Comment (pezholio) [inline server/views/applications/pages/location-factors/describe-location-factors.njk:18]: Is there a reason for this empty paragraph here?

File: server/views/applications/new.njk [hunk 2]
```diff
@@ -38,7 +38,7 @@
 
         {{ govukButton({
                     name: 'submit',
-                    text: "Submit"
+                    text: "Save and continue"
```
Comment (pezholio) [inline server/views/applications/new.njk:41]: This has bugged me for ages! Thanks for addressing it!

**Review by pezholio — APPROVED** [review]
LGTM - just one minor issue

Just as a style thing, we tend not to like long commit messages (so ones that run on - I think Github Desktop gives you a warning when you go over 50 characters), if you need to give a bit more information, summarise the changes in the subject, and then go into more detail in the description. Don't stress it for now though 👍

File: server/views/applications/pages/location-factors/describe-location-factors.njk (hunk 1 above)
Comment (beth-dixxon) [reply server/views/applications/pages/location-factors/describe-location-factors.njk:18]: That is meant to have content in it. Oh dear. I'll add it in

**Pull request merged**